variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "prefix_name" {
+    description = "The prefix name used for all resources in this environment"
+    default = "AmericanExpress2_MadhuRoy_ProjectExercise"
+}
+
+variable "location" {
+    description = "The Azure location where all resources in this deployment should be created"
+    default = "uksouth" 
+}
+
+variable "CLIENT_ID" {
```

Comment:
I don't think this is used anywhere yet

Reply:
This still isn't used anywhere; you've added more variables but you're not actually passing them through to your application inside the `app_settings` block